byon/files/seaclouds-catalog.bom

```diff
@@ -409,6 +409,77 @@ brooklyn.catalog:
             enricher.targetSensor: $brooklyn:sensor("main.uri")
             enricher.targetValue: $brooklyn:formatString("http://%s:%s", $brooklyn:attributeWhenReady("host.address"), $brooklyn:config("http.port"))
 
+  - id: seaclouds-data-collector
+    name: SeaCloudsDc
+    description: SeaClouds Data Collector 
+    item:
+      type: org.apache.brooklyn.entity.software.base.VanillaSoftwareProcess
+      id: seacloudsdc
+      name: SeaClouds Data Collector
+
+      brooklyn.config:
+        runtimeFiles:
+          https://oss.sonatype.org/content/repositories/snapshots/eu/seaclouds-project/monitor/seaclouds-data-collector/1.1.0-SNAPSHOT/seaclouds-data-collector-1.1.0-20160421.085909-4.jar : seaclouds-dc.jar
```

Comment:
This url is not available anymore. You should take the same approach as the other components. E.g:  `https://oss.sonatype.org/service/local/artifact/maven/redirect?r=snapshots&g=eu.seaclouds-project&a=planner-service&v=LATEST&e=jar`